internal/commands/user/flags.go

```diff
@@ -19,4 +19,5 @@ const (
 	flagUserShort       = "u"
 	flagUserUsage       = `set the user ids for which to filter the list of app users with`
 	flagUserDeleteUsage = `set the user ids for which to delete from the app`
+	flagUserRevokeUsage = `set the user ids for which to revoke from the app`
```

Comment:
[q] is there text from design for what should go here. In the [spec](https://docs.google.com/document/d/15Y0_jCxU2OOVx5sspQZjdewaUbFtkvWsxZUDrnkk0vc/edit#heading=h.cyglq0hem105) I saw `Select (many) from list of (filtered) app users` in the user flag prompt but I didn't see similar text in other places so wasn't sure

Reply:
flag text is not quite formalized yet... just go with mostly what you have here, just update the end to be `"...for which to revoke sessions from"`